packages/alfa-style/test/property/will-change.spec.tsx

```diff
@@ -0,0 +1,125 @@
+import { Device } from "@siteimprove/alfa-device";
+import { h } from "@siteimprove/alfa-dom";
+import { test } from "@siteimprove/alfa-test";
+
+import { Style } from "../../dist/index.js";
+
+const device = Device.standard();
+
+test("#computed parses keywords", (t) => {
+  for (let kw of ["scroll-position", "contents"] as const) {
+    const element = <div style={{ willChange: kw }} />;
+
+    const style = Style.from(element, device);
+
+    t.deepEqual(style.computed("will-change").toJSON(), {
+      value: {
+        type: "list",
+        separator: ", ",
+        values: [{ type: "keyword", value: kw }],
+      },
+      source: h.declaration("will-change", kw).toJSON(),
+    });
+  }
+});
+
+test("#computed parses custom identifiers", (t) => {
+  const value = "transform, opacity";
+  const element = <div style={{ willChange: value }} />;
+
+  const style = Style.from(element, device);
+
+  t.deepEqual(style.computed("will-change").toJSON(), {
+    value: {
+      type: "list",
+      separator: ", ",
+      values: [
+        { type: "custom-ident", value: "transform" },
+        { type: "custom-ident", value: "opacity" },
+      ],
+    },
+    source: h.declaration("will-change", value).toJSON(),
+  });
+});
+
+test("#computed parses one keyword followed by custom identifiers", (t) => {
+  for (let kw of ["scroll-position", "contents"] as const) {
+    const value = `${kw}, transform, opacity`;
+    const element = <div style={{ willChange: value }} />;
+
+    const style = Style.from(element, device);
+
+    t.deepEqual(style.computed("will-change").toJSON(), {
+      value: {
+        type: "list",
+        separator: ", ",
+        values: [
+          { type: "keyword", value: kw },
+          { type: "custom-ident", value: "transform" },
+          { type: "custom-ident", value: "opacity" },
+        ],
+      },
+      source: h.declaration("will-change", value).toJSON(),
+    });
+  }
+});
+
+test("#computed does not parse illegal custom identifiers including case permutations", (t) => {
+  for (let kw of ["inherit", "default", "will-change"]) {
+    const element = <div style={{ willChange: kw }} />;
+
+    const style = Style.from(element, device);
+
+    t.deepEqual(style.computed("will-change").toJSON(), {
+      value: {
+        type: "keyword",
+        value: "auto",
+      },
+      source: null,
+    });
+  }
+});
+
+test("#computed does not parse a valid custom ident followed by an invalid custom ident", (t) => {
+  // TODO: For some reason `source` is not `null` for these keywords.
```

Comment:
This sounds indeed very weird given that the next test does have a `null` source 🤔 

It probably means that the parser did not fail. I'd say the `List.parse` in this case does not fail but returns an empty list of animatable features?
It seems that it would make sense to constrain the list to at least 1 element (which the `List.parse` can easily do iirc).